startupscript/butane/004-git-clone-devcontainer.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# git-clone-devcontainer.sh clones a Git repository to the VM. If branch is specified, clone the specific branch.
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+function usage {
+  echo "Usage: $0 <git_url> <devcontainer_path> [branch_name]"
```

Comment:
As documented here, it implies that there are two required parameters, but the code below checks argv less than 1. Is the devcontainer_path actually a parameter?

Reply:
rm'd